ignition-server/controllers/tokensecret_controller.go

```diff
@@ -249,12 +249,17 @@ func (r *TokenSecretReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	compressedConfig := tokenSecret.Data[TokenSecretConfigKey]
 	config, err := util.DecodeAndDecompress(compressedConfig)
 	if err != nil {
+		if hasSameReasonAndMessage(tokenSecret, InvalidConfigReason, fmt.Errorf("Failed to decode and decompress config: %w", err)) {
```

Comment:
This changing from upper to lower. Error messages should start with lower case, "failed..."
Also can we put this into a var and reuse it?